### PR TITLE
Mysql default value

### PR DESCRIPTION
**Pull request by shashank-sh**

### CODE REVIEW

File: src/pyasm/search/database_impl.py [hunk 1]
```diff
@@ -3364,30 +3364,36 @@ def get_serial(self, length=4, not_null=False):
         parts.append("serial")
         return " ".join(parts)
 
-    def get_text(self, not_null=False):
+    def get_text(self, default=None, not_null=False):
```
Comment (CeltonMcGrath) [inline src/pyasm/search/database_impl.py]: Can you please add the new parameter as the last argument so that it backwards compatible?

Comment (shashank-sh) [reply src/pyasm/search/database_impl.py]: Done. Thanks for pointing it out

File: src/pyasm/search/sql.py [hunk 2]
```diff
@@ -3444,29 +3444,29 @@ def get_database(self):
         return self.db_resource.get_database()
 
 
-    def add(self, name, type, length=None, not_null=False, primary_key=False):
+    def add(self, name, type, length=None, not_null=False, primary_key=False, default=None):
         if type == "text":
-            expr = self.impl.get_text(not_null=not_null)
+            expr = self.impl.get_text(not_null=not_null, default=default)
```
Comment (CeltonMcGrath) [inline src/pyasm/search/sql.py:3449]: You need to add the new kwarg to all of the DatabaseImpl classes. (ie. PostgreSQLImp, SQLServerImpl, etc.)

Comment (shashank-sh) [reply src/pyasm/search/sql.py:3449]: I was unsure about other dbms. However, after consulting docs, I have added for other dbms as well.

Comment (shashank-sh) [reply src/pyasm/search/sql.py:3449]: Please verify again.

**Pull request closed without merging**